packages/fiori/src/IllustratedMessage.ts

```diff
@@ -273,21 +282,24 @@ class IllustratedMessage extends UI5Element {
 		return {
 			DIALOG: 679,
 			SPOT: 319,
-			BASE: 259,
+			DOT: 259,
```

Comment:
many of these values don't correspond to the visual specification. Where they come from?

Reply:
Check the **Breakpoints** table in the visual spec for Horizon. I saw that the Spot one should be 359 not 319. I'll double check with the designer.